main.py

```diff
@@ -54,7 +54,7 @@ def button_click():
 
     result = {}
 
-    if button_type == 'shuffle' or button_type == 'next':
+    if button_type == 'next' or button_type == 'shuffle' or button_type == 'back':
```

Comment:
Since we want 3/4 buttons to enter this if statement, you can do `button_type != "send"` instead